service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -63,8 +63,6 @@
 /**
  * Factory for creating Lagom service clients.
  *
- * This is designed for use from non Lagom systems.
```

Comment:
Is this not true anymore?

Reply:
Yep, I think we don't need to assume this anymore. It's all about starting a factory. It's true that there are better ways of consuming a Lagom service from another Lagom app, but there is nothing in the design that could relate to it.